src/constants/menuConstants.ts

```diff
@@ -21,8 +22,8 @@ export const MENU_ITEMS: TMenuItem[] = [
 export const MENU_ITEMS_PAUSE: TMenuItem[] = [
   { title: 'resume', route: null, action: MENU_ACTIONS.GAME_RESUME },
   { title: 'restart',  route: null, action: MENU_ACTIONS.GAME_RESTART },
-  { title: 'sound:',  route: null, action: MENU_ACTIONS.GAME_SOUND_SWITCH,
-    titleAdditionIfModifierTrue: 'on', titleAdditionIfModifierFalse: 'off' },
+  { title: 'sound',  isWithModifier: true,  route: null, action: MENU_ACTIONS.GAME_SOUND_SWITCH },
+  { title: 'vibration', isWithModifier: true, isMobileOnly: true, route: null, action: MENU_ACTIONS.GAME_VIBRATION_SWITCH },
```

Comment:
🧐
```suggestion
  { title: 'vibration', withModifier: true, mobileOnly: true, route: null, action: MENU_ACTIONS.GAME_VIBRATION_SWITCH },
```
`if (item.withModifier) ...` звучит чуть лучше, чем `if (item.isWithModifier) ...`

Reply:
ту была дилема, направшивалось withModifier, но булеан переменные мы же договорились называть начиная с is?

Reply:
хотя это не переменная тут наверное а больше как параметр

Reply:
fixed